src/extensions/core/previewAny.ts

```diff
@@ -0,0 +1,48 @@
+/*
+Preview Any - original implement from
+https://github.com/rgthree/rgthree-comfy/blob/main/py/display_any.py
+upstream requested in https://github.com/Kosinkadink/rfcs/blob/main/rfcs/0000-corenodes.md#preview-nodes
+ */
+import { IWidget } from '@comfyorg/litegraph'
+
+import { DOMWidget } from '@/scripts/domWidget'
+import { ComfyWidgets } from '@/scripts/widgets'
+import { useExtensionService } from '@/services/extensionService'
+
+useExtensionService().registerExtension({
+  name: 'Comfy.PreviewAny',
+  async beforeRegisterNodeDef(nodeType, nodeData) {
+    if (nodeData.name === 'PreviewAny') {
+      const onNodeCreated = nodeType.prototype.onNodeCreated
+
+      nodeType.prototype.onNodeCreated = function () {
+        onNodeCreated ? onNodeCreated.apply(this, []) : undefined
+
+        const showValueWidget = ComfyWidgets['STRING'](
```

Comment:
nit: Add `widget.serialize = false` for output widget to avoid the value be serialized into the workflow.

Ref: https://github.com/Comfy-Org/ComfyUI_frontend/pull/3370

Reply:
done